jobs/release-monitoring/discovery/github/Jenkinsfile

```diff
@@ -249,6 +257,45 @@ node {
         currentBuild.description = "latest: ${latestRelease}\ncurrent: ${componentRelease}"
     }
 
+    stage('Fetch Component Product Version') {
+        // Only set the product version as the latest release if the release tag var is not available
+        if (!releaseTagVar) {
+          productVersion = latestRelease
+        }
+        // Ensures that old jobs will still work with old parameter
+        // Can be removed after all service discovery jobs have been updated
+        else if (params.manifestVar) {
```

Comment:
Can't see why we would need this if we are setting the value releaseTagVar to manifestVar in the case where that's still being passed in.

Reply:
This is needed in order for the old jobs to set the product version correctly. At the moment, the `releaseTagVar` will always be defined. Manifest var will be there for the old jobs and the new webapp job will set that to a value for the release tag. If we don't catch this here, old jobs will try to go into the `else`.